spacy/errors.py

```diff
@@ -828,7 +826,6 @@ class Errors(metaclass=ErrorsWithCodes):
             "assign it a name, e.g. `@Language.component('your_name')`. You can "
             "then run `nlp.replace_pipe('{name}', 'your_name')`.")
     E969 = ("Expected string values for field '{field}', but received {types} instead. ")
-    E970 = ("Can not execute command '{str_command}'. Do you have '{tool}' installed?")
```

Comment:
This is still in use : https://github.com/explosion/spaCy/pull/12456/files#diff-edf9a5728c2b543e15658f4ab1e7ca86a63b0f18d3751b3e0c08c8f9a147083bR997